### DISCUSSION TITLE
Templates on self-hosted version

### DISCUSSION THREAD

**Discussion by cameo77** [OpnForm/OpnForm#635] [Q&A]
Hi,
if I create a form template in my self-hosted OpnForm instance, where will it be published? Only on my instance or on opnform.com? And is there a way, how I can make the template available only for my workspace users (and not public available without an account)?

**Comment by JhumanJ** [comment]
Hey! Templates are only created for your instance - not on Opnform.com. You can choose to make templates public (to your own instance) or private (only to you). It's not possible yet to create templates for the whole workspace.
